gradle.properties

```diff
@@ -28,4 +28,5 @@ org.gradle.vfs.watch=true
 android.defaults.buildfeatures.resvalues=false
 android.defaults.buildfeatures.shaders=false
 android.defaults.buildfeatures.buildconfig=true
-android.nonFinalResIds=false
\ No newline at end of file
+android.nonFinalResIds=false
+org.gradle.unsafe.configuration-cache=true
```

Comment:
Please do not change build settings in a feature PR unless required for the build to work.

Reply:
Removed all non-related changes